pallets/parachain-staking/src/lib.rs

```diff
@@ -1682,6 +1679,40 @@ pub mod pallet {
 		}
 	}
 
+	impl<T: Config> DelegatorActions<T::AccountId, BalanceOf<T>> for Pallet<T> {
+		fn delegator_bond_more(
+			delegator: &T::AccountId,
+			candidate: &T::AccountId,
+			more: BalanceOf<T>,
+		) -> DispatchResult {
+			ensure!(
+				!Self::delegation_request_revoke_exists(candidate, delegator),
+				Error::<T>::PendingDelegationRevoke
+			);
+			let mut state = <DelegatorState<T>>::get(delegator).ok_or(Error::<T>::DelegatorDNE)?;
+			state.increase_delegation::<T>(candidate.clone(), more)?;
+			Ok(())
+		}
+
+		fn testing_setup_delegator(
```

Comment:
I added this solely to be able to write benchmarks from automation-time that include staking functionality. I originally tried to feature flag this such that this method only compile while running tests but unfortunately that's not yet a supported feature- https://github.com/rust-lang/cargo/issues/2911

Open to any other patterns or ideas for achieving this.

Reply:
If its just for benchmarks you could add the benchmark compile flag. The difference being you already need to compile your code with that flag for benchmarks

Reply:
Great idea. That's exactly what I was looking for.